src/stratis_cli/_actions/_debug.py

```diff
@@ -45,7 +45,12 @@ def send_uevent(_namespace):
         """
         Issue a synthetic uevent from the CLI.
         """
-        pass  # pylint: disable=unnecessary-pass
+        sysfs_device = _namespace.device.replace("/dev/", "/sys/class/block/")
```

Comment:
I wonder if it's worth doing some simple validation checks on the device argument? It might help to give more clear error messages in the case that the user does something wrong.

E.g.:

 - Test that the passed device path exists and is a block device node
 - Verify the existence of the /sys/class/block/ directory and report an error

Another possible enhancement might be to check if the device argument is a symlink (for e.g. some `/dev/disk/by-*` link), and first try to resolve it with something like `os.path.realpath()`?

```python
>>> from os.path import realpath
>>> realpath("/dev/disk/by-uuid/a5c744e6-86d6-41de-ba87-f86bf4836242")
'/dev/nvme0n1p2'
>>> realpath("/dev/disk/by-path/pci-0000:02:00.0-nvme-1-part3")
'/dev/nvme0n1p3'
>>> 
```

Reply:
If `realpath` cannot resolve a path in the file system it returns the string partly modified (up to the point where resolution failed - or the unmodified string if nothing could be found). This may not be desirable for our use, but this can be changed by passing `strict=True`:

```
>>> realpath("/a/disk/that/doesn't/exist")
"/a/disk/that/doesn't/exist"
>>> realpath("/a/disk/that/doesn't/exist", strict=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.10/posixpath.py", line 395, in realpath
    path, ok = _joinrealpath(filename[:0], filename, strict, {})
  File "/usr/lib64/python3.10/posixpath.py", line 430, in _joinrealpath
    st = os.lstat(newpath)
FileNotFoundError: [Errno 2] No such file or directory: '/a'
```

Reply:
OK, now I see.  When I try `sudo ./bin/stratis debug uevent /dev/falsepath/disk/by-path/pci-0000\:05\:00.0-part1`, I see this:

```
    with open(sysfs_device_uevent, "w", encoding="utf-8") as uevent_file:

FileNotFoundError: [Errno 2] No such file or directory: '/sys/class/block/falsepath/disk/by-path/pci-0000:05:00.0-part1/uevent'
```

So I tried this (in a commit that I'm about to push):
```
realdevice = os.path.realpath(_namespace.device, strict=True)

FileNotFoundError: [Errno 2] No such file or directory: '/dev/falsepath'
```

Reply:
Gotcha - yes - in that case it's resolved the `/dev` but since that exists in the file system, and then left the trailing `falsepath` that it can't resolve to anything.